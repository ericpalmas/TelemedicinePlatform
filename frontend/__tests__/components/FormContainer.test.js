import React from 'react'
import renderer from 'react-test-renderer'
import FormContainter from '../../src/components/FormContainter'
import { render, screen, cleanup } from '@testing-library/react'
import 'core-js/stable'
import 'regenerator-runtime/runtime'

afterEach(cleanup)
describe('Testing message component', () => {
  it('renders', () => {
    const { asFragment } = render(<FormContainter children='test' />)
    expect(asFragment()).toMatchSnapshot()
  })
})
