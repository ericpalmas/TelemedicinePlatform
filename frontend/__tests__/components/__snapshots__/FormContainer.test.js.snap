// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing message component renders 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="justify-content-md-center row"
    >
      <div
        class="col-md-6 col-12"
      >
        test
      </div>
    </div>
  </div>
</DocumentFragment>
`;
